### ISSUE TITLE
0.1.0 interesting traceback, job left in pending state in nomad

### ISSUE THREAD

**Issue by jolexa** [jrasell/levant#162]
Hello,

I'm really not sure what happened here. Let me know what other information you might need?

```
15:18:51.263 2018/05/10 15:18:51 UTC [INFO] helper/variable: using variable with key environment and value staging from file
15:18:51.263 2018/05/10 15:18:51 UTC [INFO] helper/variable: using variable with key short_environment and value stg from file
15:18:51.263 2018/05/10 15:18:51 UTC [INFO] helper/variable: using variable with key datacenters and value [us-west-2a us-west-2b us-west-2c] from file
15:18:51.277 2018/05/10 15:18:51 UTC [INFO] levant/deploy: job nomad-log-slurp-stg not running, using template file group counts
15:18:51.277 2018/05/10 15:18:51 UTC [INFO] levant/deploy: triggering a deployment of job nomad-log-slurp-stg
15:18:52.345 2018/05/10 15:18:52 UTC [INFO] levant/deploy: evaluation a0bc064c-e231-cc2b-12d1-f1137f959419 finished successfully
15:18:52.351 2018/05/10 15:18:52 UTC [INFO] levant/job_status_checker: job nomad-log-slurp-stg has status running
15:18:54.736 2018/05/10 15:18:54 UTC [INFO] levant/job_status_checker: task nomad-log-slurp in allocation b35511f2-9cd6-0758-2d1d-a3c30b67e24c has reached running state
15:18:55.736 panic: assignment to entry in nil map
15:18:55.736 
15:18:55.736 goroutine 1 [running]:
15:18:55.736 github.com/jrasell/levant/levant.allocationStatusChecker(0xc4203e51d0, 0xc42034c000, 0x7, 0x9, 0xc420505908)
15:18:55.736 	/home/travis/gopath/src/github.com/jrasell/levant/levant/job_status_checker.go:180 +0xe8
15:18:55.736 github.com/jrasell/levant/levant.(*levantDeployment).jobAllocationChecker(0xc420322b40, 0xc420064af0, 0x13)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/levant/job_status_checker.go:107 +0x14c
15:18:55.737 github.com/jrasell/levant/levant.(*levantDeployment).jobStatusChecker(0xc420322b40, 0xc420064af0, 0x0)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/levant/job_status_checker.go:34 +0x103
15:18:55.737 github.com/jrasell/levant/levant.(*levantDeployment).deploy(0xc420322b40, 0xc420322b40)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/levant/deploy.go:179 +0x9b6
15:18:55.737 github.com/jrasell/levant/levant.TriggerDeployment(0xc4201c8a20, 0x24)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/levant/deploy.go:66 +0xb7
15:18:55.737 github.com/jrasell/levant/command.(*DeployCommand).Run(0xc42029def0, 0xc42000e0d0, 0x1, 0x5, 0xc4201bd040)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/command/deploy.go:125 +0x4ab
15:18:55.737 github.com/jrasell/levant/vendor/github.com/mitchellh/cli.(*CLI).Run(0xc4201e5900, 0xc4201e5900, 0x4, 0xc4201bd0c0)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/vendor/github.com/mitchellh/cli/cli.go:255 +0x1eb
15:18:55.737 main.RunCustom(0xc42000e080, 0x6, 0x6, 0xc42029dd70, 0xc42029dcb0)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/main.go:49 +0x433
15:18:55.737 main.Run(0xc42000e080, 0x6, 0x6, 0xb55398)
15:18:55.737 	/home/travis/gopath/src/github.com/jrasell/levant/main.go:17 +0x56
15:18:55.738 main.main()
15:18:55.738 	/home/travis/gopath/src/github.com/jrasell/levant/main.go:11 +0x63
```

**Comment by jrasell** [comment]
hey @jolexa thanks for the issue. I believe this has been fixed within https://github.com/jrasell/levant/pull/150 and so should be fixed in master and I aim to release 0.1.1 tomorrow. I will keep this ticket open to track any feedback.

**Comment by jolexa** [comment]
Ok, I'll watch for that release. Thanks.

**Comment by jrasell** [comment]
@jolexa I released 0.1.1 yesterday; if you have any feedback I would love to hear it when you have an opportunity.

**Comment by jolexa** [comment]
> I released 0.1.1 yesterday; if you have any feedback I would love to hear it when you have an opportunity.

We upgraded our staging CD server today so we will be watching for further issues. I'll try to update and close this out if I don't forget. 👍

**Comment by jolexa** [comment]
Actually, I'll just close this out now and open another report as needed!
